src/model/database/db_build.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN;
+
+CREATE TABLE users (
+    id BIGSERIAL PRIMARY KEY NOT NULL,
+    username TEXT NOT NULL,
```

Comment:
I think this should be VARCHAR(64) with a limit on length so that we can display this in our nav bar ok.